File: src/app/classes/Point.ts
export class Point {
    x: number;
    y: number;
    constructor(x: number, y: number){
        this.x = x;
        this.y = y;
    }
    shift(p: Point){
        this.x += p.x;
        this.y += p.y;
    }
}

File: src/main/java/com/PaintApp/PaintAppBackend/Shapes/Point.java
package com.PaintApp.PaintAppBackend.Shapes;

public class Point {
    public int x;
    public int y;
    Point(int x, int y){
        this.x = x;
        this.y = y;
    }
    void shift(Point p){
        this.x += p.x;
        this.y += p.y;
    }
}
